Client/Cliqz/Frontend/Browser/QuerySuggestionView.swift

```diff
@@ -49,11 +49,13 @@ class QuerySuggestionView: UIView {
     }
     
     func didEnterText(text: String) {
-        guard QuerySuggestions.isEnabled() else {
+        currentText = text
+        
+        guard QuerySuggestions.isEnabled() && UIDeviceOrientationIsPortrait(UIDevice.currentDevice().orientation) else {
```

Comment:
UIDevice.currentDevice().orientation has a problem. Your device can be in portrait, but face up on a desk. This method will return face up, even though it you are in portrait mode. Instead use OrientationUtil
isPortrait(). This compares the width of the screen to the height in order to determine the orientation.